report/generator/flow-report-assets.js

```diff
@@ -9,12 +9,13 @@ const fs = require('fs');
 
 /* eslint-disable max-len */
 const FLOW_REPORT_TEMPLATE = fs.readFileSync(`${__dirname}/../../flow-report/assets/standalone-flow-template.html`, 'utf8');
+const REGULAR_REPORT_CSS = fs.readFileSync(__dirname + '/../assets/styles.css', 'utf8');
 const FLOW_REPORT_CSS = fs.readFileSync(`${__dirname}/../../flow-report/assets/styles.css`, 'utf8');
 const FLOW_REPORT_JAVASCRIPT = fs.readFileSync(`${__dirname}/../../dist/report/flow.js`, 'utf8');
 /* eslint-enable max-len */
 
 module.exports = {
   FLOW_REPORT_TEMPLATE,
-  FLOW_REPORT_CSS,
+  FLOW_REPORT_CSS: [REGULAR_REPORT_CSS, FLOW_REPORT_CSS].join('\n'),
```

Comment:
I'm fine with this approach for now, but it's doing the same thing as https://github.com/GoogleChrome/lighthouse/pull/13260 which adds regular report styles via the inlined JS component.